modules/vaos/app/services/vaos/jwt.rb

```diff
@@ -2,6 +2,13 @@
 
 module VAOS
   class JWT
```

Comment:
Please add top-level class documentation for [YARDocs](https://www.rubydoc.info/github/department-of-veterans-affairs/vets-api

Reply:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/3689